### PR TITLE
Updated pi to <username> and formatting

### PR DESCRIPTION
**Pull request by jchiarulli**
Updated pi to <username>, NOTE formatting, and general formatting for the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,18 +10,35 @@ You will need:
...
-[Raspberry Pi Imager](https://www.raspberrypi.com/software/) is the recommended way to install Raspberry Pi OS. 
+[Raspberry Pi OS Docs](https://www.raspberrypi.com/documentation/computers/getting-started.html)
+[Raspberry Pi Imager](https://www.raspberrypi.com/software/) is the recommended way to install Raspberry Pi OS.
 
 ```
 sudo apt install rpi-imager
 ```
 
-I recommend installing the `Raspberry Pi OS (64-bit)` for now you can find that in the *other* section.
+I recommend installing the `Raspberry Pi OS (64-bit)` for now you can find that in the _other_ section.
```
Comment (ChristianChiarulli) [inline README.md:41]: my formatter prefers ** what formatter are you using?

File: README.md [hunk 2]
```diff
@@ -160,7 +177,7 @@ All of the dependencies can be found in the `build-unix.md` file located in the
 - Build requirements:
 
 ```
-sudo apt install build-essential libtool autotools-dev automake pkg-config bsdmainutils
+sudo apt install build-essential libtool autotools-dev automake pkg-config bsdmainutils python3
```
Comment (ChristianChiarulli) [inline README.md:180]: doesn't raspbian ship with python3, if not why would it be a build requirement?

Comment (jchiarulli) [reply README.md:180]: I think python3 comes pre-installed with raspian, but someone may be using a different OS that doesn't come with python3. Unless we're only making a guide for raspian?

Also, python3 is included in the bitcoin-core build-unix.md file here: https://github.com/bitcoin/bitcoin/blob/master/doc/build-unix.md#linux-distribution-specific-instructions

File: README.md (hunk 1 above)
Comment (jchiarulli) [reply README.md:41]: I'm using prettier. Do you have a preferred formatter?

**Pull request closed without merging**